### PR TITLE
Using TestNG

### PR DESCRIPTION
**Pull request by HaykMelikyan**
Write calculator program and write tests for it.

### CODE REVIEW

File: src/main/java/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        return a - b;
+    }
+
+    public static double multiplication(double a, double b) {
+        return a * b;
+    }
+
+    public static double division(double a, double b) {
+        Double B = b;
+        if (B.equals(0.0)) {
```
Comment (chandranik90) [inline src/main/java/Calculator.java:17]: `Double.compare()` could be a better solution, without initializing an extra object.

File: src/test/java/CalculatorTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    @Test(description = "Division negative test")
+    public void divisionNegTest() {
+        double actualResult = Calculator.division(15.3, 2.8);
+        double expectedResult = 15.3 / 2.8 + 1;
+        Assert.assertNotEquals(actualResult, expectedResult);
+    }
+
+    // Addition test with assertTrue method
```
Comment (chandranik90) [inline src/test/java/CalculatorTest.java:70]: By the way, you may use JavaDoc style comments if you want to have a description for a method.

**Review by chandranik90 — APPROVED** [review]

File: src/main/java/Calculator.java (hunk 1 above)
Comment (HaykMelikyan) [reply src/main/java/Calculator.java:17]: `Double.comare()` returns int, but I wanted to get boolean value for that condition statement.

Comment (chandranik90) [reply src/main/java/Calculator.java:17]: ok

File: src/test/java/CalculatorTest.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    @Test(description = "Addition positive test")
+    public void additionBoolTest() {
+        Double actualResult = Calculator.addition(15.3, 2.8);
+        Double expectedResult = 15.3 + 2.8;
+        Assert.assertTrue(actualResult.equals(expectedResult));
```
Comment (HamTorosyan) [inline src/test/java/CalculatorTest.java:75]: Why to not use `assertEqual()` on line 75?

**Review by HamTorosyan — APPROVED** [review]
For the future, please keep some branch naming convention, "Using TestNG" is not so meaningful.

File: src/test/java/CalculatorTest.java (hunk 3 above)
Comment (HaykMelikyan) [reply src/test/java/CalculatorTest.java:75]: One of the tasks was the use of the `assertTrue()` or `assertFalse()` method, so I wrote the same addition testing method, but with a boolean check.

**Pull request merged**